Core/src/GrpcTrait.php

```diff
@@ -273,4 +269,29 @@ private function formatTimestampForApi($value)
             'nanos' => (int) $nanos
         ];
     }
+
+    /**
+     * Return a custom API endpoint, if one is provided by the user.
+     *
+     * @param array $config
+     * @return string|null
+     */
+    private function getApiEndpoint(array $config)
+    {
+        return isset($config['apiEndpoint']) && $config['apiEndpoint']
```

Comment:
It seems like most usage of this is wrapped by an isset check already, do we need this method?